app/views/request/_after_actions.html.erb

```diff
@@ -49,7 +49,7 @@
         <li class="action-menu__menu__item">
           <ul class="action-menu__menu__submenu anyone_actions">
             <li>
-              <%= link_to _("Report this request"), new_request_report_path(:request_id => @info_request.url_title) %>
+              <%= link_to _("Report this request"), new_request_report_path(:request_id => @info_request.url_title) %><span class="action-menu__info-link"><%= link_to _("Help"), help_about_path %></span>
```

Comment:
I'm not sure where this should go?

`<%= link_to _("Help"), help_about_path(:anchor => "focused") %>` maybe?